docs/content/en/v2/development/styling.md

```diff
@@ -0,0 +1,375 @@
+
+You might want to refactor and extract that style to be able to reuse it across your app.  
+Fortunately, Fabulous has you covered.
+
+When you want to apply the same group of modifiers on each widgets of a same type, you can write your own shorthand modifier:
+
+```fs
+open System.Runtime.CompilerServices
+
+[<Extension>]
+type SharedStyle =
+    [<Extension>]
+    static member inline largeText(this: WidgetBuilder<'msg, #ILabel>) =
+        this
+            .font(namedSize = NamedSize.Large, fontFamily = "Helvetica")
+            .horizontalTextAlignment(TextAlignment.End)
+            .textColor(Color.Blue.ToFabColor())
+            .padding(10.)
+
+// Usage
+Label("Hello")
+    .horizontalTextAlignment(TextAlignment.Start)
+    .largeText()
+
+Label("World")
+    .horizontalTextAlignment(TextAlignment.End)
+    .largeText()
+    .backgroundColor(Color.Red.ToFabColor())
+```
+
+As you can see, the `largeText` modifier will automatically apply the font size, the font family, the text alignment, the text color and the padding to all `Label` widgets where it's applied.  
+In addition to this shorthand modifier, you can still use all available modifiers on the same widget.
+
+### How to write a shorthand modifier
+
+Writing a shorthand modifier can be overwhelming the first time.  
+It is making use of several advanced features of F#. Let's take a step back and decompose the code.
+
+First thing to understand is that a shorthand modifier is actually implemented as an [extension method](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/type-extensions).
+
+To write an extension method, you need to have the following basic structure:
+
+```fs
+// Opening this namespace is required to use the Extension attribute below
+open System.Runtime.CompilerServices
+
+// No constructor is needed for this type. There will only be static methods declared inside.
+[<Extension>]
+type SomeTypeHoldingExtensionMethods =
+    // The first parameter has to be the type we want to extend
+    [<Extension>]
+    static member SomeExtensionMethod(this: TypeForWhichWeWantToAddAnExtension) =
+        // here we do something with the parameter this
+        this
+```
+
+Let's start by adding the keyword `inline` before the name of the method. This will let Fabulous optimize the code for better performances at runtime.
+
+```fs
+[<Extension>]
+static member inline SomeExtensionMethod(this: TypeForWhichWeWantToAddAnExtension) =
+```
+
+**Now, the most important part to understand:** when using a widget, Fabulous will actually give you an instance of `WidgetBuilder<'msg, 'marker>` where `'marker` will be a type representing the widget.  
+For example, `Label()` will give you `WidgetBuilder<'msg, ILabel>`,
+
+You might have noticed that in the code above, we are using `#ILabel` instead of `ILabel`.  
+The `#` (pound) symbol lets F# know that we actually want the shorthand modifier to be available to all widgets deriving from `ILabel`, not just `Label`.
```

Comment:
Should we add a link the the Flexible types feature ? https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/flexible-types

Reply:
Good idea! Didn't even know that it was called a flexible type 😅